airflow/providers/dingding/hooks/dingding.py

```diff
@@ -49,20 +51,20 @@ class DingdingHook(HttpHook):
     def __init__(
         self,
         dingding_conn_id='dingding_default',
-        message_type='text',
-        message=None,
-        at_mobiles=None,
-        at_all=False,
+        message_type: str = 'text',
+        message: Union[str, dict, None] = None,
```

Comment:
```suggestion
        message: Optional[Union[str, dict]] = None,
```
We mark optional arguments in this way. I think it's worth keeping one style.